index.html

```diff
@@ -318,6 +318,31 @@
       </section>
       <section>
         <h3>
+          Application's icon
+        </h3>
+        <p>
+          The application icon is derived from the <a><code>icons</code> member</a>
+          array, depending on the size of the image resources and other meta data
+          such as the <a>purpose member</a>.
+        </p>
+        <p>
+          <dfn>Adaptive icons</dfn> are icons which are prepared to support masks and
```

Comment:
No, it should read "are icons which X prepends to support masks" - who is X? the browser, the OS, the developer? Otherwise, it's not clear who has the responsibility to do that.